pymc3/__init__.py

```diff
@@ -31,14 +31,29 @@
         _log.addHandler(handler)
 
 
-if not semver.match(theano.__version__, ">=1.1.2"):
-    print(
-        "!" * 60
-        + f"\nThe installed Theano(-PyMC) version ({theano.__version__}) does not match the PyMC3 requirements."
-        + "\nFor PyMC3 to work, Theano must be uninstalled and replaced with Theano-PyMC."
-        + "\nSee https://github.com/pymc-devs/pymc3/wiki for installation instructions.\n"
-        + "!" * 60
-    )
+def _check_backend_version():
+    backend_paths = theano.__spec__.submodule_search_locations
+    try:
+        backend_version = theano.__version__
+    except:
```

Comment:
Should we print the exception in case that helps figuring out different issues under the same umbrella?

Reply:
The exception in this case is just `AttributeError: module 'theano' has no attribute '__version__'` as seen in https://github.com/pymc-devs/pymc3/issues/4510#issuecomment-802721737

I don't think that would help in any way. And the conclusion would still be the same..